internal/binutils/binutils_test.go

```diff
@@ -401,3 +414,105 @@ func TestOpenMalformedMachO(t *testing.T) {
 		t.Errorf("Open: got %v, want error containing 'Mach-O'", err)
 	}
 }
+
+func TestObjdumpVersionChecks(t *testing.T) {
+	// Test that the objdump version strings are parsed properly.
+
+	var version string
+	if runtime.GOOS == "darwin" {
+		// Valid Apple LLVM version string with usable version.
+		version = `Apple LLVM version 11.0.3 (clang-1103.0.32.62)
+		Optimized build.
+		Default target: x86_64-apple-darwin19.4.0
+		Host CPU: skylake`
+		if runtime.GOOS == "darwin" && !isLLVMObjdump(version) {
```

Comment:
runtime.GOOS == "darwin" is already checked above, this check is redundant.

Reply:
Done.